### DISCUSSION TITLE
🔗 Title and Caption underneath gallery or Portfolio image (Duplicate of #11378)

### DISCUSSION THREAD

**Discussion by garaiburu** [elementor/elementor#24358] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

There is no way to position titles and/or captions underneath images in galleries or portfolios

### Describe the solution you'd like

It would be nice if you could position titles and/or captions underneath images in galleries and portfolio's, rather than in an overlay on top.

**Comment by nicholaszein** [comment]
This request is a duplicate of #11378 and will be merged with it after it is migrated.

**Comment by garaiburu** [comment]
I see that there's been no action taken on this for the last 4 years. Does that mean it will never happen?
